src/App.svelte

```diff
@@ -1,372 +1,26 @@
-								</ToolTip>
-							</td>
-						</tr>
-					{/each}
-				{:else}
-					{#each completedTasks as task}
-						<tr class:done={task.done}>
-							<td>
-								<ToolTip title="Click to toggle whether this task has been completed.">
-									<button on:click={completeTask(task)}><Emoji symbol="🏆" label="Completed"></Emoji></button>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to edit this task.">
-									<td contenteditable="true" bind:innerHTML={task.text}/>
-								</ToolTip>
-							</td>
-							<td>
-								<ToolTip title="Click to delete this task from the list.">
-									<button on:click={() => deleteTask(task)}><Emoji symbol="🗑️" label="Delete"></Emoji></button>
-								</ToolTip>
-							</td>
-						</tr>
-					{/each}
-				{/if}
-			</table>
-		</div>
-
-		<footer>
-			<div class="counter">
-				{#if count === 1}
-					{count} task remaining
-				{:else}
-					{count} tasks remaining
-				{/if}
-			</div>
-			<div class="spacer">|</div>
-			<div class="mode">
-				Showing {mode} tasks
-			</div>
-			<div class="spacer">|</div>
-			<div class="filterAll">
-				<button on:click={() => filterAllTasks()}>All</button>
-			</div>
-			<div class="spacer">|</div>
-			<div class="filterActive">
-				<button on:click={() => filterActiveTasks()}>Active</button>
-			</div>
-			<div class="spacer">|</div>
-			<div class="filterCompleted">
-				<button on:click={() => filterCompletedTasks()}>Completed</button>
-			</div>
-			<div class="spacer">|</div>
-			<div class="clear">
-				<button on:click={() => clearAllTasks()}>Clear All Tasks</button>
-			</div>
-			<div class="spacer">|</div>
-			<div class="toggle">
-				<button on:click={() => toggleAlerts()}>Toggle Alerts</button>
-			</div>
-			
-		</footer>
-	</main>
-</body>
-
-<style>
-	body {
-		background-image: url('https://upload.wikimedia.org/wikipedia/commons/f/f3/43228_1_miscellaneous_digital_art_simple_simple_patterns.jpg');
-		background-size: cover;
-		text-align:center;
-	}
-
-	* {
-		background-color: transparent;
-		color: white;
-	}
-
-	main {
-		text-align: center;
-		padding: 1em;
-		margin: 0 auto;
-		max-width: 100%;
-	}
-
-	h1 {
-		color: white;
-		font-size: 5em;
-		font-weight: 400;
-		font-family:'Courier New', Courier, monospace;
-	}
-
-	h2 {
-		align-items: center;
-		color: white;
-		font-size: 1.5em;
-		font-family:'Courier New', Courier, monospace;
-		text-align: center;
-		padding: 1em;
-		max-width: 100%;
-		margin: 0 auto;
-	}
-
-	table.main {
-		background-color: transparent;
-		font-family: 'Courier New', Courier, monospace;
-		font-size: 1.5em;
-		max-width: 100%;
-	}
-
-	.pane {
-		display: inline-block;
-    	overflow-y: auto;
-		overflow-x: hidden;
-    	max-height: 500px;
-		max-width: 100%;
-	}
+<Header />
 
-	.centered {
-		margin-left:auto; 
-    	margin-right:auto;
-	}
+<main>
+  <TaskList {alertsOn} />
+</main>
 
-	.italic {
-		font-style: italic;
-	}
+<Footer bind:alertsOn />
```

Comment:
For the record, the semantic HTML structure for top-level page layout is:

```html
<header></header>
<main></main>
<footer></footer>
```

Previously, you had everything in `<main>`, which is no bueno.